scripts/coronaviruskoreadataset.json

```diff
@@ -0,0 +1,342 @@
+{
+    "archived": "zip",
+    "citation": "",
+    "description": "Dataset of COVID-19 in South Korea",
+    "homepage": "https://github.com/jihoo-kim/Coronavirus-Dataset",
+    "keywords": [
+        "Github",
+        "Coronavirus",
+        "Covid-19",
+        "Machine Learning"
+    ],
+    "licenses": [
+        {
+            "name": "CC BY-NC-SA 4.0"
+        }
+    ],
+    "name": "coronaviruskoreadataset",
+    "resources": [
+        {
+            "dialect": {
+                "delimiter": ","
+            },
+            "name": "case",
+            "path": "case.csv",
+            "schema": {
+                "fields": [
+                    {
+                        "name": "case_id",
+                        "type": "int"
+                    },
+                    {
+                        "name": "province",
+                        "size": "17",
+                        "type": "char"
+                    },
+                    {
+                        "name": "city",
+                        "size": "15",
+                        "type": "char"
+                    },
+                    {
+                        "name": "grp",
+                        "size": "5",
+                        "type": "char"
+                    },
+                    {
+                        "name": "infection_case",
+                        "size": "37",
+                        "type": "char"
+                    },
+                    {
+                        "name": "confirmed",
+                        "type": "int"
+                    },
+                    {
+                        "name": "latitude",
+                        "size": "9",
+                        "type": "char"
+                    },
+                    {
+                        "name": "longitude",
+                        "size": "10",
+                        "type": "char"
+                    }
```

Comment:
`latitude, longitude` should not be char

Reply:
Fixed it :+1: